src/components/Dashboard/DashboardCallout.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { ReactNode } from "react";
+import {
+  Popup,
+  Button,
+  MoreIcon,
+  ComponentEventHandler,
+  PopupProps,
+  Menu,
+  mergeThemes,
+  ComponentVariablesInput,
+  Provider as FluentUIThemeProvider,
+  ThemePrepared,
+  EyeSlashIcon,
+} from "@fluentui/react-northstar";
+
+export interface IWidgetActionKey {
+  id: string;
+  icon?: JSX.Element;
+  title: string;
+}
+
+export interface IDashboardCallout {
+  open: boolean;
+  onOpenChange: ComponentEventHandler<PopupProps>;
+  menuProps: any;
+  globalTheme: ThemePrepared;
```

Comment:
This looks like a normal component, why is it using the local theme pattern?

Reply:
Please, check comment below